apple/imac/14-2/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+{
+  imports = [
+    ../.
+    ../../../common/gpu/intel
+    ../../../common/gpu/nvidia
+    ../../../common/hidpi.nix
+  ];
+
+  boot = {
+    initrd.availableKernelModules = [
+      "nvme"
+      "sd_mod"
+      "thunderbolt"
+      "usb_storage"
+      "xhci_pci"
+    ];
```

Comment:
We do not include code that can is generated by nixos-generate-config in nixos-hardware

```suggestion
```